HRD/HRD_multiqc_config_v1.0.0.yml

```
@@ -0,0 +1,141 @@
+# See multiqc/utils/config_defaults.yaml for the defaults.
+module_order:
+    - fastqc:
+        module_tag:
+            - DNA
+    - bcl2fastq:
+        module_tag:
+            - DNA
+
+table_columns_visible:
+    FastQC:
+        avg_sequence_length: True
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)